COPYRIGHT_TRANSFER

```diff
@@ -0,0 +1 @@
+I, Seth Ladd, hereby transfer copyright of dart_to_js_script_rewriter to Workiva Inc.
```

Comment:
Sorry about this, I realize that in the example I sent you it said `transfer` but could you update that to be `assign`